### PR TITLE
Removed the ability for hxCodec and hxvlc to be in the same project

### PR DESCRIPTION
**Pull request by EgguMan**
Its quite simple, in Handle.hx under init it checks if the project has HxCodec. If the project has hxCodec, then it will not allow you to compile.

I've seen many people, myself include, get very unclear bugs caused by HxCodec being in the same project as hxvlc. This would save many people much time.

### CODE REVIEW

File: source/hxvlc/util/Handle.hx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package hxvlc.util;
 
 #if (!cpp && !(desktop || mobile))
-#error 'The current target platform isn\'t supported by hxvlc.'
+// #error 'The current target platform isn\'t supported by hxvlc.'
```
Comment (MAJigsaw77) [inline source/hxvlc/util/Handle.hx]: Why?

**Comment by NeeEoo** [comment]
Why not

```hx
#if hxCodec
#error "hxvlc and hxCodec cannot be used in the same project."
#end
```

**Comment by mcagabe19** [comment]
+1

**Comment by moxie-coder** [comment]
> Why not
> 
> ```hx
> #if hxCodec
> #error "hxvlc and hxCodec cannot be used in the same project."
> #end
> ```

yeah that's the reason why this PR is pretty much useless

**Comment by EgguMan** [comment]
Edited it to follow the sugestions 👍🏻

**Review by moxie-coder — APPROVED** [review]
that's more like it

**Review by MAJigsaw77 — COMMENTED** [review]
Why not place it on top of the class?

File: source/hxvlc/util/Handle.hx (hunk 1 above)
Comment (EgguMan) [reply source/hxvlc/util/Handle.hx]: Was an accident from testing - would not recognize the platform on my local branch, so the linter wouldnt work.

**Pull request merged**